hack/release.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+BUILD_DATE="$(TZ=Etc/UTC date +%Y%m%d)"
+RELEASE_DIR="${RELEASE_DIR:-nightly/${BUILD_DATE}}"
+
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_SCM_HASH="$(git rev-parse HEAD)"
+
+TMPDIR="$(mktemp -d /tmp/crrelXXXXXXXX)"
+function clean_up() {
+  if [[ "${TMPDIR}" == "/tmp/crrel"* ]]; then
+    rm -rf "${TMPDIR}"
+  fi
+}
+trap clean_up EXIT
+
+cd "${SCRIPT_ROOT}/.."
+
+# Build copy the tools.
+bazel build //cmd/crinit //cmd/clusterregistry
+
+# Create the archives.
+pushd bazel-bin/cmd/crinit
+tar czf "${TMPDIR}/clusterregistry-client.tar.gz" crinit
+popd
+pushd bazel-bin/cmd/clusterregistry
+tar czf "${TMPDIR}/clusterregistry-server.tar.gz" clusterregistry
+popd
+
+pushd "${TMPDIR}"
+
+# Create the SHAs.
+sha256sum clusterregistry-client.tar.gz > clusterregistry-client.tar.gz.sha
```

Comment:
Same here. Except use absolute paths?

Reply:
In this case, I don't think that's possible. sha256sum generates a SHA with the full path to the file if you provide it. So, if you run `sha256gen /foo/bar`, the output SHA will reference that folder and will fail to verify.